pages/about.html

```diff
@@ -4,9 +4,58 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About | Julie Warren</title>
+  <link rel="stylesheet" href="../style.css" />
+  <!-- favicon -->
+  <link rel="apple-touch-icon" sizes="180x180" href="../assets/favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="../assets/favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="../assets/favicon/favicon-16x16.png">
+  <link rel="manifest" href="../assets/favicon/site.webmanifest">
+  <link rel="mask-icon" href="../assets/favicon/safari-pinned-tab.svg" color="#087e8b">
+  <meta name="msapplication-TileColor" content="#2b5797">
+  <meta name="theme-color" content="#ffffff">
 </head>
 <body>
-  
+  <!-- navigation bar -->
+  <header class="about-header">
+    <nav class="nav-container">
+      <a href="../index.html">
+        <img id="nav-logo" alt="logo of initials J A W enclosed in a circle" src="../assets/images/JAW.png">
+      </a>
+      <ul class="nav-buttons">
+        <li>
+          <a href="../index.html">Home</a>
+        </li>
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a href="./portfolio.html">Portfolio</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="break">
+    </div>      
+    <h1 class="page-header">
+        About Julie
+    </h1>
+  </header>
+  <!-- main body -->
+  <main class="main-container">
+    <div class="about-container">
+      <img id=j-bicycle alt="Julie standing with her touring bicycle" src="../assets/images/jbike.jpg">
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Sed turpis tincidunt id aliquet risus feugiat in. Quam vulputate dignissim suspendisse in est ante. Dolor purus non enim praesent elementum. Interdum consectetur libero id faucibus nisl. Sit amet consectetur adipiscing elit ut aliquam purus. A condimentum vitae sapien pellentesque habitant morbi tristique senectus. Pulvinar elementum integer enim neque volutpat ac tincidunt. 
+        <br><br>
+        Cras semper auctor neque vitae tempus. Amet est placerat in egestas. Vulputate sapien nec sagittis aliquam malesuada bibendum arcu vitae elementum. Vitae proin sagittis nisl rhoncus mattis rhoncus urna neque. Tortor condimentum lacinia quis vel. Nibh tortor id aliquet lectus proin nibh nisl.
+        <br><br>
+        Non odio euismod lacinia at. Morbi non arcu risus quis varius quam quisque. Nulla facilisi cras fermentum odio eu feugiat. Cras fermentum odio eu feugiat pretium. Donec adipiscing tristique risus nec feugiat in fermentum posuere. Fames ac turpis egestas maecenas pharetra convallis posuere. Lobortis feugiat vivamus at augue eget. Consectetur a erat nam at lectus. Tincidunt ornare massa eget egestas. At augue eget arcu dictum.
+      </p>
```

Comment:
Ditto about the use of `<br>` here. These are three different paragraphs, so they should each be wrapped in a `<p>`.